_careers/data.md

```diff
@@ -0,0 +1,140 @@
+- Prediction and clasification: we have already built some initial models to predict the number of offers a tender will receive. We want to do many other things in this front. Predict prices, discounts, detect possible frauds...
+- Entity extraction: we have tons of PDFs with unstructured content (companies, concepts, contract clauses...). Once we have extracted them, many important products can be built.
+
+And many other challenges that will appear along the way. We'll start bit by bit and grow the team which you may end up managing (if that's something you'd want, that is).
+
+We use Jenkins to automate tasks. We are a Ruby On Rails shop, so some Ruby knowledge would be ideal, but we understand you'll be using Python and the usual suspects.
+
+You'll also be contributing from time to time to Gobierto Transparency, our open source solution for transparency and accountability initiatives.
+
+We work on [Github](http://github.com/populatetools/) using a git workflow with CI/CD, tests, different automations... Our workflow has been polished throughout a few years, so have automated many things (monitoring, error tracking, server setups & updates, deployments, various checks...). You can take a look at the [Gobierto repo to see how we actually work](http://github.com/populatetools/gobierto).
+
+
+## Day to day work
+
+- **Plan** new features of Gobierto and existing applications, and contribute to new apps from scratch
+- **Implement** defined features, manage your sprints, do sprint reviews
+- **Collaborate** with other team members: provide feedback and do code reviews
+- **Improve** and maintain existing code of production applications
+
+
+## Your first week
+
+(in no particular order)
+
+- Meet the team!
+- Set up your tools, install Gobierto
+- Browse our internal doc wiki to check out how we work, where to find things, etc.
+- Browse around in our general issue tracker (Github)
+- Get assigned initial issues to navigate around our development process: tests, deploy, code reviews...
+
+
+## Your first quarter
+
+- Master all of our internal tools and processes
+- You are tasked with planning a medium-sized feature in Gobierto that will take you several weeks to implement
+- Propose the work plan and work with a designer and a front-end engineer to implement it
+- Deploy code to staging, demo your work to the team/client, write documentation, polish it and merge it to master
+
+
+## Your first year
+
+- Responsible of one or more apps/modules/big features
+- Master Gobierto and have provided ideas to improve code and features
+- Speak with clients autonomously if needed
+- Do fluent code reviews
+- Have made suggestions to improve our processes
+- Have attended some conferences and proposed books to our book club
+- Given internal talks about tech and non-tech things
+- Have traveled with the rest of the team to one or more retreats (if global pandemics allow it)
```

Comment:
- Responsible of data in Populate
- Our workflows and pipelines are more mature
- Creating new data projects is a smooth process
- Data analysis has been used in different projects, not only Gobierto Contratos
- There are different ML models deployed

Reply:
just change the things :D